experimental/python/databricks/bundles/jobs/_models/power_bi_table.py

```diff
@@ -12,9 +12,7 @@
 
 @dataclass(kw_only=True)
 class PowerBiTable:
-    """
-    :meta private: [EXPERIMENTAL]
-    """
+    """"""
```

Comment:
@kanterov Can you take a look at why this yields an empty doc block?

Reply:
@pietern it works as intended, class doesn't have experimental annotation and no description in JSON schema

Reply:
Shouldn't empty docstrings be omitted from the output instead?

Reply:
From what I remember missing class doc string breaks the parsing of field doc strings

Reply:
I'm not sure it's still needed. I there was a problem with VSCode, but I have found that it's resolved now. Databricks SDK doesn't add empty doc strings, at least we should keep both consistent, I will create a PR to remove empty doc strings.